test/jdk/java/awt/print/PrinterJob/PrintGlyphVectorTest.java

```diff
@@ -97,54 +115,42 @@ public int print(Graphics g, PageFormat pf, int pageIndex) {
         return Printable.PAGE_EXISTS;
     }
 
+    private static void createTestUI() {
+        Frame f = new Frame("Test PrintGlyphVector");
+        PrintGlyphVectorTest pvt = new PrintGlyphVectorTest();
+        f.add(pvt, BorderLayout.CENTER);
+
+        Button printButton = new Button("PRINT");
+        printButton.addActionListener((e) -> {
+            PrinterJob pj = PrinterJob.getPrinterJob();
+            if (pj == null || pj.getPrintService() == null ||
+                    !pj.printDialog()) {
+                return;
+            }
+            pj.setPrintable(new PrintGlyphVectorTest());
+            try {
+                pj.print();
```

Comment:
Does it show Print dialog? Should the test fail if the user clicks Cancel there?

Any exception thrown should also fail the test, we expect no exceptions.